inverse_velocity/loss_landscape.py

```diff
@@ -0,0 +1,106 @@
+import time
+import sys
+import os
+import numpy as np
+import json
+import pickle
+import torch.utils.checkpoint
+
+sys.path.append("/work2/08264/baagee/frontera/gns-main/")
+from example.inverse_velocity.forward import rollout_with_checkpointing
+
+from gns import reading_utils
+from gns import data_loader
+from gns import train
+
+
+# Work dir
+path = "data/"
```

Comment:
If you are expecting it in `data/` either make sure the script does this or specify in the README